server/src/entity/measure.ts

```diff
@@ -0,0 +1,29 @@
+import { Ingredient } from '@entity';
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from './baseEntity';
+
+@Entity()
+export class Measure extends BaseEntity<Measure> {
+    @PrimaryGeneratedColumn()
+    public id: number;
+
+    @Column()
+    public amount: number;
```

Comment:
amount can differ per unit, lbs vs grams for example. Check the docs or TokenScope for an example of nested types:
https://typeorm.io/#/embedded-entities